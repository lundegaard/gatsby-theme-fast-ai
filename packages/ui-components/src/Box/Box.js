import React from 'react';
import { Box as RebassBox } from 'rebass';

const Box = props => <RebassBox {...props} />;

export default Box;
