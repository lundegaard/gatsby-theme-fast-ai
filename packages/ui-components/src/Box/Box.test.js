import React from 'react';
import { shallow } from 'enzyme';

import Box from './Box';

describe('<Box />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<Box {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
