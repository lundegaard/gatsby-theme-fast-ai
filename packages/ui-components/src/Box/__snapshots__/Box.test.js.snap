// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> matches snapshot when shallowly rendered 1`] = `<Styled(div) />`;
